bors.toml

```diff
@@ -3,7 +3,7 @@
 pr_status = [
  "python-tests",
  "tools-build-success",
- "check-labels",
+# "check-labels",
```

Comment:
Care to add a comment why this was disabled? (just in case reverting this takes longer than expected again..)